testCourse/questions/addNumbers/question.html

```diff
@@ -4,3 +4,9 @@
 </pl-question-panel>
 
 <pl-number-input answers-name="c" comparison="sigfig" digits="3" label="$c=$"></pl-number-input>
+
+<pl-hidden-hints>
+  <pl-hint show-after=3>
+    Try counting on your fingers!
+  </pl-hint>
+</pl-hidden-hints>
```

Comment:
It would be nice to have an example question in the `exampleCourse` too. Most people don't look in the test course, so they won't be able to discover the element.

You should also add it to the documentation in `docs/elements.md`. It would probably go in the "Conditional" elements category.

Reply:
Will add examples + docs once some more comments come in and the behavior is finalized. I'm also wondering whether it's worth getting rid of the "backend mode", since client questions can always just generate the HTML through the mustache template.